src/View/Helper/PaginatorHelper.php

```diff
@@ -1121,4 +1121,42 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Dropdown array for Select limit
+     *
+     * @param array $limits This is array of options.
+     * @param int $default Default limit for option selecting. Default is Paginator limit.
+     * @param array $options Options for Select tag attributes like class, id or event
+     *
+     * @return string html output.
+     */
+    public function limitSelect($limits = [], $default = null, $options = [])
+    {
+        if (!in_array('Form', $this->helpers)) {
+            return "";
+        }
+
+        $out = $this->Form->create(null, ['type' => 'get']);
+
+        if (empty($limits)) {
+            $limits = [10 => $this->Number->format(10), 25 => $this->Number->format(25), 50 => $this->Number->format(50), 100 => $this->Number->format(100)];
+        }
+
+        if (empty($default) || !is_integer($default)) {
+            $default = $this->param('perPage');
+        }
+
+        if (!in_array($default, $limits)) {
+            $limits += [$default => $this->Number->format($default)];
+        }
+
+        ksort($limits);
+
+        $out .= $this->Form->input('limit', ($options + ['type' => 'select', 'label' => __("View") . '&nbsp;', 'escape' => false, 'value' => $default, 'id' => 'select_limit', 'default' => $default, 'options' => $limits, 'onChange' => 'this.form.submit()']));
```

Comment:
Thank you. It has no use actually. I used it in my project and thus forgot to remove. Thank you. Its removed.